src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,10 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder res = new StringBuilder();
```

Comment:
let's create more understandable name of the variable instead of `res`

Reply:
result would be fine? or binary would be better?

Reply:
done

Reply:
`binary` is an adjective
variables name should be a noun

Reply:
so `result` is better